heartbeat/azure-events.in

```diff
@@ -221,7 +222,7 @@ class clusterHelper:
 
 		nodes = []
 		nodeList = clusterHelper._exec("crm_node", "--list")
```

Comment:
I now that this code is inherited, but `nodeList` doesn't follow the python naming conventions.
It should be `node_list` or `nodelist`

Reply:
No need to change it here though, as it follows the convention of the code.

Your other suggestions seems good though.